Makefile

```diff
@@ -189,7 +189,7 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
 	operator-sdk bundle validate ./bundle
 
 .PHONY: bundle-build
-bundle-build: ## Build the bundle image.
+bundle-build: bundle ## Build the bundle image.
```

Comment:
This seems like an unrelated change?

Reply:
Kind of unrelated to the $TOPIC, but I added it as a separate commit.

Reply:
It seemed too simple of a change to create a new MR for it, but if you like it can be done separately.

Reply:
I'll be sending this change with a new MR then. Thanks